python/cudf/cudf/io/parquet.py

```diff
@@ -577,11 +577,26 @@ def read_parquet(
         )
     filepath_or_buffer = paths if paths else filepath_or_buffer
 
+    # Prepare remote-IO options
+    prefetch_options = kwargs.pop("prefetch_options", {})
```

Comment:
Do we need to document this somewhere? I don't see `prefetch_options` anywhere in any public APIs.

Reply:
Yeah, Sorry about that. I added some info to the `read_parquet` and `get_reader_filepath_or_buffer` doc-strings. I marked this  arguments as 'experimental' for now, because I don't really want users to treat it as "public" yet (we may modify the UX a bit when kvikio is added to the picture).